src/ruby/qps/proxy-worker.rb

```diff
@@ -143,7 +165,7 @@ def proxymain
   # Configure any errors with client or server child threads to surface
   Thread.abort_on_exception = true
 
-  s = GRPC::RpcServer.new
+  s = GRPC::RpcServer.new(pool_size: 1024, max_waiting_requests: 1024)
```

Comment:
Why did you need to override these parameters? Can you add a comment explaining why?

Reply:
`DEFAULT_POOL_SIZE = 30` and `DEFAULT_MAX_WAITING_REQUESTS = 20`,
when ruby proxy create more than 30 php client, the connection fails.

Reply:
The setting of `pool_size` to allow more `DEFAULT_POOL_SIZE` concurrent calls than makes.

`max_waiting_requests` shouldn't be needed though - that arg gets ignored.

Reply:
Done.